src/Microsoft.PowerShell.Commands.Utility/commands/utility/Import-LocalizedData.cs

```diff
@@ -217,7 +217,7 @@ protected override void ProcessRecord()
                         {
                             // Mark untrusted values for assignments to 'Global:' variables, and 'Script:' variables in
                             // a module scope, if it's necessary.
-                            ExecutionContext.MarkObjectAsUntrustedForVariableAssignment(variable, scope, Context.EngineSessionState);
+                            System.Management.Automation.ExecutionContext.MarkObjectAsUntrustedForVariableAssignment(variable, scope, Context.EngineSessionState);
```

Comment:
Oh, maybe ask .Net team rename new System.Threading.ExecutionContext to System.Threading.ThreadExecutionContext before they release 6.0?

Reply:
[Recommendation ](https://github.com/dotnet/runtime/issues/57017#issuecomment-894618406) is
```
 <DisableImplicitNamespaceImports>true</DisableImplicitNamespaceImports>
<DisableImplicitNamespaceImports_DotNet>true</DisableImplicitNamespaceImports_DotNet>
```
and remove the workaround after Preview7.

Reply:
I will add this workaround and file an issue to remove it after releases after preview 7, as it will be disabled.